cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -6,42 +6,21 @@
 from subprocess import check_output
 
 from charms import layer
-from charms.docker import DockerOpts
```

Comment:
+1 to removing docker configuration from this top most layer. Moving that into the container runtime layer was a great win today.